packages/java/hilla-bom/pom.xml

```diff
@@ -4,6 +4,13 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
+    <parent>
+        <groupId>com.vaadin</groupId>
+        <artifactId>hilla-project</artifactId>
+        <version>24.5-SNAPSHOT</version>
+        <relativePath>../../../pom.xml</relativePath>
+    </parent>
+
     <groupId>com.vaadin</groupId>
```

Comment:
```suggestion
```

Reply:
i am going to check if there are other modules has the duplicated version.. 
for example https://github.com/vaadin/hilla/blob/main/packages/java/hilla/pom.xml#L18

but they are not a blocker, i think